### PR TITLE
Docs reset password

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Docs reset password
## Reference Requirements
- Task ID: #194

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: docs/api-references/reset_password.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "message": "The given data was invalid.",
+    "errors": {
+        "email": [
+            "The email must be a valid email address."
+        ]
+    }
+}
+```
+
+### `PUT` Resetpassword
```
Comment (at-vietpham) [inline docs/api-references/reset_password.md]: **New password**

File: docs/api-references/reset_password.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```
+/api/password/reset
+```
+Put password new
```
Comment (at-vietpham) [inline docs/api-references/reset_password.md]: New password

File: docs/api-references/reset_password.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
+## Send Mail Resetpassword Api
+
+### `POST` Send Mail Resetpassword
```
Comment (at-vietpham) [inline docs/api-references/reset_password.md]: Reset password

File: docs/api-references/reset_password.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+## Send Mail Resetpassword Api
```
Comment (at-vietpham) [inline docs/api-references/reset_password.md]: Reset password

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-vietpham** [comment]
LGTM 🍔

**Pull request merged**